### ISSUE TITLE
Improve interface clarity and friendliness

### ISSUE THREAD

**Issue by CKPEIIKA** [quantum-entangled/machine-learning-ui#52]
1. **Training Status Display** Display the current training status: loaded dataset, model, whether it has been trained or not. Currently, when something is loaded, you can only determine it by its content or the respective tab.
2. **Multiple Models Support** Add support for using multiple models or change the behavior of the "create model" button. Currently, it simply erases the previous model without warning.
3. **Default Names** Use default names for input layers and models when the "name" field is empty, such as "model_dataset_name_N" and "input_1." This eliminates the need for users to specify each layer name.
4. **Tooltip and Documentation Links**: Provide a tooltip or hint when certain options are grayed out, such as when users can choose an output or set loss functions. This tooltip can be accompanied by a link to more detailed documentation.
5. **Reasonable Defaults and Advice** Use reasonable defaults whenever possible, like a default test set of 30%, and offer guidance for making appropriate selections.
I can try to work on some of these suggestions when learn the new interface architecture better if it is reasonable.

**Comment by quantum-entangled** [comment]
Thanks for your suggestions! I have a few comments:

1. Do you mean the status of the data and model? Because the training status is displayed in the respective tab while training. As for data and models - yes, that would be nice, I just haven't figured out the best way to do it yet.
2. Supporting multiple models would complicate things a lot. When the model is uploaded, Streamlit displays a badge with uploading status, and there you can remove some files. However, it disappears after changing tabs. Probably, we can figure out how to display it there for users so they know they will erase the model if they upload a new one.
3. That is a tricky thing. It's good if you don't care about your layers' names, but if you do, you have to remove these defaults each time, which is not ideal too. I would say that in most cases, you should explicitly name your layers with meaningful information (e.g., "N2_molecule_input"), which comes directly from your data, so we can't predefine it. The "input_1", "input_2" stuff is just for illustrative purposes.
4. For now, there are several "info boxes" with information about grayed-out zones. Do you mean to change them or something else?
5. Sure, in terms of different hyperparameters, there are the provided defaults; we just need to use them everywhere.

**Comment by quantum-entangled** [comment]
1. If you mean seeing, for example, that the training process is running while being on some other page - unfortunately, it's not possible with Streamlit because when you leave the page, it stops the processes on it and refreshes it when you come back.
2. That seems like a workable approach, I suppose.
3. Yes, I wanted to provide more explicit comments in each section to make it more intuitive. We can achieve this not just with badges but also with markdown comments. For badges, we need to properly adjust the events to display them correctly.
